adit/upload/processors.py

```diff
@@ -0,0 +1,17 @@
+from adit.core.models import DicomTask
+from adit.core.processors import TransferTaskProcessor
+
+from .models import (
+    UploadSettings,
+    UploadTask,
+)
+
+
+class UploadTaskProcessor(TransferTaskProcessor):
```

Comment:
What's the purpose of this processor?! It seems that the upload is directly handled in the async view.